src/index.html

```diff
@@ -5,8 +5,350 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,300&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <span>
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="img-logo"
+        >
+      </span>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple
+            </a>
```

Comment:
Перенось текст на новий рядок, тоді простіше буде його прочитати
```suggestion
            <a href="#" class="nav__link is-active">
              Apple
            </a>
```